src/test/kotlin/no/nav/dagpenger/inntekt/v1/UklassifisertInntektApiTest.kt

```diff
@@ -113,39 +113,6 @@ class UklassifisertInntektApiTest {
         }
     }
 
-    @Test
-    fun `POST unknown uklassifisert inntekt should return 404 not found`() = testApp {
```

Comment:
noen grunn for å fjerne disse?

Reply:
Ja, jeg fjernet endepunktene de testet (vi bytta ut post med get for å hente inntekter, men beholdt post-endepunktene en stund for å ikke brekke apiet)

Reply:
aha!